### ISSUE TITLE
0px left margin when viewed in medium-width windows

### ISSUE THREAD

**Issue by wslack** [18F/hub#202]
Saw in Safari and FFX that the left margin on any hub page disappears if the window is too skinny; however, margins snap in at very low width (ie: mobile).

**Comment by mbland** [comment]
Yep, it's known that the site isn't terribly responsive yet. As we get more interest/resources, we'll spend more effort on these issues before we transition from alpha to beta.
